web/src/components/footer.js

```diff
@@ -96,6 +99,9 @@ export default function Footer() {
                         <LogoLink to="/">
                             <TextLogoWhite />
                         </LogoLink>
+                        <StyledLink href="mailto:info@hubble.net">
```

Comment:
I think you should add `target="_blank" rel="noreferrer"` so this opens in a new page